executor/index_merge_reader.go

```diff
@@ -719,6 +721,13 @@ func (w *indexMergeProcessWorker) fetchLoop(ctx context.Context, fetchCh <-chan
 
 	distinctHandles := make(map[int64]*kv.HandleMap)
 	for task := range fetchCh {
+		if task.err != nil {
+			// Tell main routine to finish. And continue drain fetchCh.
+			task.doneCh = make(chan error, 1)
+			task.doneCh <- task.err
+			resultCh <- task
+			continue
```

Comment:
Should we stop processing the remained task? 
Continue or break or something else?

Reply:
done